src/relay-runtime/BUILD.bazel

```diff
@@ -0,0 +1,14 @@
+load("//:bazel/macros/build_yarn_workspace.bzl", "build_yarn_workspace")
+load("//:bazel/macros/test_yarn_workspace.bzl", "test_yarn_workspace")
+
+build_yarn_workspace(
+    name = "relay-runtime",
+    package_name = "@adeira/relay-runtime",
+)
+
+test_yarn_workspace(
+    name = "test",
+    deps = [
+        "@universe//src/js",
```

Comment:
This makes the source code of adeira/js available, and the test will transpile it?

Would it make sense to use the build target instead?

Reply:
It could be like this eventually, I was thinking about it as well. But now I am trying to keep it as close as possible to what we have now.

Reply:
Sure, we can always improve later 😊